### PR TITLE
Remove Quarkus prefix from docs

### PR DESCRIPTION
**Pull request by gastaldi**
You can try it by running in your cloned repository:

	jbang run ../quarkiverse-devops/patches/AdjustDocTitle.java

### CODE REVIEW

File: patches/AdjustDocTitle.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+            options.setIndent(3);
+            options.setIndicatorIndent(2);
+            options.setDefaultFlowStyle(DumperOptions.FlowStyle.BLOCK);
+            Yaml yaml = new Yaml(options);
+            Map<String, String> data = Collections.emptyMap();
+            try (var is = new FileInputStream(antoraFile)) {
+                data = yaml.load(is);
+            }
+            String title = data.get("title");
+            if (title.startsWith("Quarkus ")) {
```
Comment (gsmet) [inline patches/AdjustDocTitle.java:28]: You need to handle `Quarkus - Java Embedded Framework` first.

And could we also drop the ` Extensions for Quarkus` from `CXF Extensions for Quarkus`?

File: patches/AdjustDocTitle.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+                data.put("title", title.substring("Quarkus ".length()));
```
Comment (gsmet) [inline patches/AdjustDocTitle.java:29]: Let's trim it just in case.

**Review by gsmet — COMMENTED** [review]
I suggested two improvements.

**Comment by holly-cummins** [comment]
Another option, but it's probably not the right one to suggest at this point, is that we pass the name through to antora from maven, as asciidoc variable. That's DRYer. 

The end effect is the same, though.

File: patches/AdjustDocTitle.java (hunk 1 above)
Comment (gastaldi) [reply patches/AdjustDocTitle.java:28]: Because they are specific, it's easier to create those PRs manually for this case

**Comment by gastaldi** [comment]
@holly-cummins good idea, we can do it in the same patch AFAIK. Can you perform that change in a PR from any quarkiverse repo so I can mimic it?

**Comment by gastaldi** [comment]
> Another option, but it's probably not the right one to suggest at this point, is that we pass the name through to antora from maven, as asciidoc variable. That's DRYer.

On a second thought, let's perform the change in a separate PR, as some Quarkiverse repositories handle attributes differently (eg. https://github.com/quarkiverse/quarkus-cxf/blob/e65737ff6a49bf784bbd120fbd256ba98d72112e/docs/antora.yml#L7-L25)

**Pull request merged**